tests/cypress/e2e/unit_tests/rbac_fleet.spec.ts

```diff
@@ -62,4 +65,90 @@ describe('Test Fleet access with RBAC with custom roles', { tags: '@rbac' }, ()
     })
   )
 
+  qase(43,
+    it('Test "Std-Base" user with custom role "list" and "create" to "fleetworkspaces" can list and create workspaces but NOT  "edit" nor "delete" them', { tags: '@fleet-43' }, () => {
```

Comment:
Please use full name i.e. 
`'Test 'Standard Base' Role user with "list" and "create" verbs for "fleetworkspaces" resource. User can NOT "edit" or "delete" them'`